modules/reference/pages/properties/broker-properties.adoc

```diff
@@ -338,6 +397,22 @@ The Schema Registry provides configuration properties to help you enable produce
 
 For information on how to edit broker properties for the Schema Registry, see xref:manage:cluster-maintenance/node-property-configuration.adoc[].
 
+=== mode_mutability
+
+Allow modifying mode.
```

Comment:
Not sure what this means--allows the mode to be modified? More info/guidance might help users here.